apps/storybook/docs/d_planning/1_historiek-en-toekomst.stories.mdx

```diff
@@ -0,0 +1,135 @@
+import {Meta} from '@storybook/blocks';
+import planning2023 from '../../resources/planning/planning-2023.png';
+import planning2024 from '../../resources/planning/planning-2024.png';
+import planning2025 from '../../resources/planning/planning-2025.png';
+
+<Meta title="Planning/Historiek & Toekomst"/>
+
+# Historiek & Toekomst
+
+## Inhoudstafel
+
+- [Aanpak](#aanpak)
+- [Nu: Planning 2024](#nu-planning-2024)
+- [Toekomst: Planning 2025](#toekomst-planning-2025)
+- [Verleden: Q4 2023 / Q1 2024](#verleden-q4-2023--q1-2024)
+- [Historiek](#historiek)
+
+
+## Aanpak
+
+Het UIG-team heeft altijd als doel gehad de drempel tot frontend ontwikkeling te verlagen. Enerzijds gebeurd
+softwareontwikkeling bij Departement Omgeving typisch door fullstack-ontwikkelaars die praktisch meer affiniteit hebben
+met backend- dan met frontend-ontwikkeling. Anderzijds is het de bedoeling dat designs de
+[huisstijl](https://www.vlaanderen.be/vlaanderen-intern-communicatie/huisstijl-en-merkbeleid) - voorzien door [Digitaal
+Vlaanderen](https://www.vlaanderen.be/digitaal-vlaanderen) - gebruiken. Deze huisstijl-voor-websites wordt praktisch
+gecapteerd door het [Webuniversum](https://overheid.vlaanderen.be/webuniversum/v3/documentation/) in de vorm van
+componenten. Dit zijn echter geen web-componenten, het is hoofdzakelijk css styling (technisch scss),
```

Comment:
Ik zou hier wel nog verduidelijken wat bedoeld word met web-components (door bv. te linken naar https://developer.mozilla.org/en-US/docs/Web/API/Web_components zoals je deed in de Introductie).

En eventueel de zin ook nog aanpassen naar: `Dit zijn echter geen web-componenten zoals gedefiniëerd door het W3C...`

Gezien Web Universum zijn eigen library ook "Webcomponents" noemt, is het goed om zo duidelijk mogelijk te zijn.

Reply:
Ok, ik maak ervan:
Dit zijn echter geen [web-componenten](https://developer.mozilla.org/en-US/docs/Web/API/Web_components),
die in de browser geregistreerd kunnen worden. DV noemt het wel 'webcomponents', maar het is hoofdzakelijk css styling (technisch scss), met beperkt Javascript die gedocumenteerd worden a.d.h.v. voorbeelden.